security/security_profiles_operator/spo-release-notes.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="spo-release-notes"]
+= Security Profiles Operator release notes
+include::_attributes/common-attributes.adoc[]
+:context: spo-release-notes
+
+toc::[]
+
+The Security Profiles Operator provides a way to define secure computing (https://kubernetes.io/docs/tutorials/security/seccomp/[seccomp]) and SELinux profiles as Custom Resources, synchronizing profiles to every node in a given namespace.
+
+These release notes track the development of the Security Profiles Operator in {product-title}.
+
+For an overview of the Security Profiles Operator, see xref:../security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
+
+== Security Profiles Operator 0.5.0
+
+The following advisory is available for Security Profiles Operator 0.5.0:
+
+* link:https://access.redhat.com/errata/RHBA-2022:TODO[RHBA-2022:TODO - OpenShift Security Profiles Operator bug fix update]
```

Comment:
Hopefully the errata would be created tomorrow (all the required metadata was created today)

Reply:
Here's the errata link: https://errata.devel.redhat.com/advisory/106256

Reply:
Thank you! I'll approve the docs and update the PR.